eos-updater/tests/meson.build

```diff
@@ -0,0 +1,12 @@
+# Copyright 2022 Endless OS Foundation, LLC
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+# python-dbusmock doesn’t currently provide a way to install out-of-tree templates
+
+# lint check
+test(
+  'flake8',
+  py3,
+  args: ['-m', 'flake8', files('eos_updater.py')],
```

Comment:
Might we want to check for the presence of flake8? Two ways to do this:

1. Invoke it via its wrapper
```
test(
  'flake8',
  find_program('flake8'),
  args: [files('eos_updater.py')],
  ...
)
```

2. One can use the `python3` module to find a python3 installation that has the desired modules (`flake8` and `dbusmock` here I guess)

Reply:
`python-flake8` is listed in `Build-Depends` in `debian/control`, and this meson snippet is the same as the other two uses of `flake8` in the source tree.

Using `find_program()` might make meson fail a little earlier if it’s not found, but that would require installing the [`flake8`](https://packages.debian.org/bullseye/all/flake8/filelist) package as well as [`python3-flake8`](https://packages.debian.org/bullseye/all/python3-flake8/filelist) (the latter only provides the module). So that seems a little over the top.

I’ll look at adding a commit to search for the module, which will at least make meson fail a bit more gracefully when building it without Debian packaging on a system which doesn’t have `python3-flake8` installed.

Reply:
OK, I didn't realise it's already used in this project.

Reply:
I’ve added an additional commit which checks for it when finding the Python installation.